chronos/asyncfutures2.nim

```diff
@@ -60,6 +60,10 @@ type
       closure*: iterator(f: Future[T]): FutureBase {.raises: [Defect, CatchableError, Exception], gcsafe.}
     value: T ## Stored value
 
+  # Future with a tuple of possible exception types
```

Comment:
document that this should not be used generally and may change in the future

Reply:
Done